descriptions/rep/pt_br.lua

```diff
@@ -364,6 +364,7 @@ local repTrinkets={
 	[142] = {"142","","No início de cada andar, gera 4 Book of Virtues {{Collectible584}} orbitais"}, --  Beth's Faith
 	[143] = {"143","","Impede que o item carregue#Chance de gerar uma lil Battery quando limpa uma sala"}, --  Old Capacitor
 	[144] = {"144","","Lágrimas que erram os inimigos irão ser redirecionar 90 graus em direção aos inimigos"}, --  Brain Worm
+	[145] = {"145","","↑ +10 Sorte enquanto segurar#É destruída quando receber dano"}, -- Perfection
```

Comment:
@wofsauge This item is not present in English, I don't know if it was intentional, but even so I translated